ee/modules/610-service-with-healthchecks/hooks/ensure_crds.go

```diff
@@ -0,0 +1,12 @@
+/*
+Copyright 2024 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package hooks
+
+import (
+	"github.com/deckhouse/deckhouse/go_lib/hooks/ensure_crds"
+)
+
+var _ = ensure_crds.RegisterEnsureCRDsHook("/deckhouse/modules/610-service-with-healthchecks/crds/*.yaml")
```

Comment:
we don't have to define crd ensuring hooks for modules anymore, addon-operator does it [automatically](https://github.com/flant/addon-operator/pull/497)

Reply:
Thanks. Done